task_5/Anna.Kononenko/IntSream1.java

```diff
@@ -0,0 +1,16 @@
+package task1;
+
+import java.util.stream.IntStream;
+
+/**1. �������� ���������, ��������� �� ����� ��� �������������
+ * ����� ������������������ 1000 1003 1006 1009 1012 1015 �.
+ * Created by ANN on 22.12.2016.
```

Comment:
Created by ... information and date is saved in git repository, so no need for this comment